qualimap/qualimap.cwl

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+# Metadata
+label: qualimap-qc
+doc: This is qualimap CWL tool definition http://qualimap.bioinfo.cipf.es/
+
+hints:
+  ResourceRequirement:
+    ramMin: 4000
+    coresMin: 1
+  SoftwareRequirement:
+    packages:
+      qualimap:
+        version: [ "2.2.2d" ]
+        specs:
+          - https://identifiers.org/rrid/RRID:SCR_001209
+          - https://bio.tools/qualimap
+  DockerRequirement:
+    dockerPull: 'quay.io/biocontainers/qualimap:2.2.2d--1'
+requirements:
+  - class: ShellCommandRequirement
+
+# Base command
+baseCommand: [ "qualimap"]
+
+# Perform RNA-seq QC analysis on paired-end data http://qualimap.bioinfo.cipf.es/doc_html/command_line.html
```

Comment:
This should go in the `doc` so that those using a GUI to interact with this tool definition can see that

Reply:
Thanks, done.